rllib/models/tests/test_distributions.py

```diff
@@ -94,8 +94,7 @@ def test_categorical(self):
 
         inputs = inputs_space.sample()
 
-        for fw, sess in framework_iterator(
-                session=True, frameworks=("tf", "tf2", "torch")):
+        for fw, sess in framework_iterator(session=True):
```

Comment:
Assuming that by default it iterates through all frameworks, I think there are a lot more tests that need to have the same deletion to reflect this update.

Reply:
Probably. But this is a start :) Ideally, we should test with all frameworks everywhere.

Reply:
I mean the currently existing tests that use all frameworks, they should be replaced with this new line of code ^^

Reply:
Yes, you are right. Actually, I'm not even sure why I made these fixes above. Not really related to this PR. We should do a separate PR, where we comb through all occurrences of this and fix those.